src/engine/platform/opl.cpp

```diff
@@ -1590,7 +1599,7 @@ void DivPlatformOPL::reset() {
   }
   */
   if (downsample) {
-    const unsigned int downsampledRate=(unsigned int)((double)rate*rate/chipRateBase);
+    const unsigned int downsampledRate=(unsigned int)((double)rate*49716.0/(double)chipRateBase);
```

Comment:
Use `COLOR_NTSC/72.0` for exact value (it's 49715.9090909...}

You may round the final result.